src/knw/KNWRobot.h

```diff
@@ -703,7 +746,7 @@ class KNWRobot
          * }
          * @endcode
          */
-     bool setupServo(int id, int pin);
+     bool setupServo(int id, int pin, int zero = 90);

```

Comment:
Oh wait, I see it defaulted here. Then as a follow-up, you'll need to add an `@param zero` explaining what that variable means in the docs right above it. Do that for any function where you changed the signature.